doc/classes/SceneTree.xml

```diff
@@ -225,7 +225,7 @@
 			For mobile platforms, see [member quit_on_go_back].
 		</member>
 		<member name="current_scene" type="Node" setter="set_current_scene" getter="get_current_scene">
-			The current scene.
+			Returns the top node of the current running scene, regardless of its composition.
```

Comment:
```suggestion
			Returns the root node of the current running scene, regardless of its composition.
```
"Top" is confusing as it can mean many things, and AFAIK we don't use that term to denote this elsewhere, "root" however is regularly used across the documentation

Reply:
The top node is the best term here. Because a `root` member already exists, but it's not the `top node`, it's the `Window`.

So `get_tree().get_node("/root")` will not return the "top node" directly. For that you will need this: `get_tree().get_node("/root").get_child(0)`.

I've updated the description by adding an example.

Reply:
I agree with @AThousandShips, the term should be "root". The sentence specifically talks about the "root node of the current scene", which is plenty of qualifiers to indicate a very specific thing. There is no "top node" terminology in the engine in the sense that you apply here. It can be confused with the visual order very easily too.

The example should probably be put in a code block. It should also avoid using undefined variables, such as `node_path`. A literal string would work better here. Also you are using the `[param foo]` syntax incorrectly. It's a reference tag, it is validated against parameters of methods and signals. Which is not the case here.

Reply:
I would like to use the term "root", but as I explained to @AThousandShips, the `root` member already exists, and is the `-1` element of the tree:

`get_node("/root/").get_index() returns -1`

So I will update the commit, changing from "top node" to "main node" and delete the example part. I believe the description is good enough. Is this ok?

"Returns the main node of the current running scene, regardless of its structure."

Reply:
That's also not an established terminology, the term is "root", and as Yuri said, it's clear from "root node **of the current scene**" (emphasis mine)

Reply:
But that will conflict, no? Asserting the `current_scene` is equivalent to the "root" node while there is already a member named root?

Reply:
It won't, it is the *root node of the current scene*, the `root` node *isn't* part of the current scene

Also this terminology is already established:
https://docs.godotengine.org/en/latest/classes/class_nodepath.html#description

There it uses "root node" of the main scene

Reply:
@AThousandShips Disregard my last message. I understood that there is no conflict. 

What about "Returns the active scene’s root node, regardless of its structure."?

Reply:
I'd use "current" to not introduce a new term that doesn't really apply, "active" isn't really a thing

Reply:
Ok. Changed it to:

"Returns the current running scene root node, regardless of its structure."

Reply:
Why complicate it? What's wrong with:
"Returns the root node of the current scene, regardless of its structure."

Also that sounds ungrammatical to me, shouldn't it be "currently running"

Reply:
Ok. Changed it again:

Returns the root node of the current scene, regardless of its structure.

Reply:
I would also add:
```
[b]Warning:[/b] Setting this directly might not work as expected, and will [i]not[/i] add or remove any nodes from the tree, consider using [method change_scene_to_file] or [method change_scene_to_packed] instead.
```
As assigning to this is low level and requires for example that the node already be in the scene and be a child of `root` (or null)